operators/nvidia_video_codec/nv_video_writer/metadata.json

```diff
@@ -0,0 +1,56 @@
+{
+	"operator": {
+		"name": "NVIDIA Video Codec SDK: Video File Writer",
+		"description": "This operator writes H.264/H.265 video files from raw encoded frames.",
+		"authors": [
+			{
+				"name": "Holoscan Team",
+				"affiliation": "NVIDIA"
+			}
+		],
+		"language": ["C++"],
```

Comment:
```suggestion
		"language": ["C++", "Python"],
```

Reply:
I've updated all four related operators.

Reply:
Thanks, I missed this in the previous PR